libGraphite/data/data.cpp

```diff
@@ -320,16 +307,12 @@ auto graphite::data::block::increase_size_to(std::size_t new_size) -> void
 
 auto graphite::data::block::clear() -> void
 {
-    set((uint32_t)0);
+    set((uint8_t)0, size());
 }
 
 auto graphite::data::block::set(uint8_t value, std::size_t bytes, block::position start) -> void
 {
-    union simd::value v;
-    for (unsigned char & byte : v.bytes) {
-        byte = value;
-    }
-    simd::set(get<std::uint32_t *>(start), size() - start, v, bytes);
+    memset(get<std::uint8_t *>(start), value, bytes);
 }
```

Comment:
I'll need to verify that the use of memset and memcpy here is faster. In theory they should be, but earlier versions of these algorithms from the start of the year were using them, and I updated to the current implementations for performance improvements.

I'm currently adding testing into the library and will be adding some metrics to it as well to time measurements, so I'll want to run this through that as well to be sure the performance is acceptable. Due to the size of the assets being used by Kestrel, slight discrepancies can add up quickly.

Reply:
Cross platform optimisations are tricky, huh? 😂
All I can say is this is all working much faster for me on macOS/x86_64, but I realise it could be different on other platforms.